discovery-provider/docker-compose.yml

```diff
@@ -389,6 +390,19 @@ services:
       - /var/k8s/bolt:/bolt
 
   # plugins
+
+  ddex:
+    image: audius/ddex:${TAG:-2109fdf8676d9925514f41d186f47ea36fa03643}
+    container_name: ddex 
+    restart: unless-stopped
+    entrypoint: "./scripts/docker-entrypoint.sh"
```

Comment:
with changes from https://github.com/AudiusProject/audius-protocol/pull/7013 we should remove this line